js_modules/dagster-ui/packages/ui-core/src/graph/asyncGraphLayout.ts

```diff
@@ -36,13 +36,18 @@ const asyncGetFullOpLayout = asyncMemoize((ops: ILayoutOp[], opts: LayoutOpGraph
 const _assetLayoutCacheKey = (graphData: GraphData) => {
   // Note: The "show secondary edges" toggle means that we need a cache key that incorporates
   // both the displayed nodes and the displayed edges.
-  return JSON.stringify(graphData);
+  return JSON.stringify({
```

Comment:
I changed this to exclude the definitions of nodes since that wouldn't affect the layout

Reply:
Hmm `getAssetNodeDimensions` can return different values based on the node data (if a node has partitions, the box is larger) but it seems acceptable to assume that that won't change. Or at least, if you add a tag to your asset and they're sorta "too close together" for a while, it's not the end of the world.